script/var/init/storage.sh

```diff
@@ -42,7 +42,10 @@ BIND_EMULATOR() {
 }
 
 # Drastic
-BIND_EMULATOR save/drastic/backup drastic/backup
+#BIND_EMULATOR save/drastic/backup drastic/backup
+
+# Drastic
+BIND_EMULATOR save/drastic/backup drastic-trngaje/backup
```

Comment:
If I understood @trngaje correctly, there's backup directory redirection support again (via settings.json?), so we may not need this bind mount anymore. 🤞

Reply:
From what I can see, we can still only redirect save states. I didn't see anything obvious to redirect saves.